src/SkyApm.Diagnostics.FreeRedis/FreeRedisTracingDiagnosticProcessor.cs

```diff
@@ -0,0 +1,65 @@
+﻿using FreeRedis;
+using SkyApm;
+using SkyApm.Common;
+using SkyApm.Config;
+using SkyApm.Diagnostics;
+using SkyApm.Tracing;
+using SkyApm.Tracing.Segments;
+using System;
+using System.Linq;
+
+namespace SkyApm.Diagnostics.FreeRedis
+{
+
+
+    /// <summary>
+    /// FreeRedisTracingDiagnosticProcessor
+    /// </summary>
+    public class FreeRedisTracingDiagnosticProcessor : ITracingDiagnosticProcessor
+    {
+        public static readonly StringOrIntValue Free_Redis = new StringOrIntValue(7, "Redis");
```

Comment:
Redis is a server-side ID, you should update the main repository to register a new ID with mapping

- https://github.com/apache/skywalking/blob/master/oap-server/server-starter/src/main/resources/component-libraries.yml#L135
- https://github.com/apache/skywalking/blob/96b6690e24fb306dde309922da70bc3bf16fa28d/oap-server/server-starter/src/main/resources/component-libraries.yml#L725